odoo/addons/base/models/res_company.py

```diff
@@ -158,6 +161,12 @@ def _compute_logo_web(self):
             img = company.partner_id.image_1920
             company.logo_web = img and base64.b64encode(tools.image_process(base64.b64decode(img), size=(180, 0)))
 
+    @api.depends('partner_id.image_1920', 'logo')
```

Comment:
As it is not stored, you may normally use only 'logo' which already depends on partner_id.image_1920